exercises/exercise-07/index.ts

```diff
@@ -41,7 +41,7 @@ interface Admin {
     role: string;
 }
 
-type PowerUser = unknown;
+type PowerUser = Omit<User, 'type'> & Omit<Admin, 'type'> & {type: 'powerUser'};
```

Comment:
It wasn't clear to me what you were supposed to do in this exercise if you _didn't_ do the "higher difficulty" version.